src/Target.h

```diff
@@ -258,6 +258,16 @@ struct Target {
     /** Was libHalide compiled with support for this target? */
     EXPORT bool supported() const;
 
+    /** Given a size (in bytes), return True if the allocation size can fit
```

Comment:
I'd also like to rename this to indicate that it is not establishing a certainty of the allocation being on the stack, but determining if a certain size might ever be allocated on the stack.